dat/dictionary.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
We can merge the files into a valid `sample.json` but then sample.json would correspond to a nested object. It would be difficult to test `lists` and `objects` separately while keeping them in a single file.